.vscode/settings.json

```
@@ -19,5 +19,6 @@
     "python.analysis.extraPaths": [
         "./src"
     ],
-    "python.analysis.typeCheckingMode": "basic"
+    "python.analysis.typeCheckingMode": "basic",
+    "python-envs.pythonProjects": []
```

```suggestion
    "python.analysis.typeCheckingMode": "basic"
```

nit: Remove unrelated change

---

Not sure how this ended up in the PR, but it has now been removed.